tests/test_diagnostics.py

```diff
@@ -0,0 +1,59 @@
+import pyspark.sql.functions as f
+import pandas as pd
+
+from splink.diagnostics import _splink_score_histogram
+import pytest
+
+
+def test_score_hist_adjusted(spark, gamma_settings_4, params_4, sqlite_con_4):
+
+    """
+    test that a dataframe gets processed when function is given a column name to take splink score from
```

Comment:
Once we've implemented probability density could we have a test that checks that it's computed correctly.

e.g. put explicitly define a dataframe with `tf_adjusted_match_prob` = [0.1, 0.3, 0.5, 0.7, 0.9] and make sure that the probability density is the correct value (possibly 0.2?) with all 5 bins